### ISSUE TITLE
cli: pipeline install error

### ISSUE THREAD

**Issue by FeelyChau** [alibaba/pipcook#535]
When I run `pipcook pipeline install ./examples/pipeline/text-bayes-classification.json`, it looks like the plugins not installed.
```shell
$ pipcook pipeline install ./example/pipelines/text-bayes-classification.json
ℹ start to create pipeline
✔ pipeline is created: 2hlhup34
ℹ check plugin:
✔ pipeline installed successfully.
✖ @pipcook/plugins-csv-data-collect undefined.
✖ @pipcook/plugins-csv-data-access undefined.
✖ @pipcook/plugins-bayesian-model-define undefined.
✖ @pipcook/plugins-bayesian-model-train undefined.
✖ @pipcook/plugins-bayesian-model-evaluate undefined.
```

**Comment by FeelyChau** [comment]
I checked it again, the latest version on NPM has this bug, master has fixed.
We should close this issue after 1.1.3 released.

**Comment by FeelyChau** [comment]
It's fixed by v1.2.0. #551
